### PR TITLE
feat: add `gcp_compute_instance_group_manager` table

### PR DESCRIPTION
**Pull request by pdecat**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
> select
  m.name as manager_name, m.target_size, m.status -> 'isStable' as is_stable,
  g.name as group_name,
  g.size as group_size
from
  gcp_compute_instance_group_manager as m,
  gcp_compute_instance_group as g
where
  m.instance_group ->> 'name' = g.name;
+----------------------------------------------------+-------------+-----------+----------------------------------------------------+------------+
| manager_name                                       | target_size | is_stable | group_name                                         | group_size |
+----------------------------------------------------+-------------+-----------+----------------------------------------------------+------------+
| gke-project1234567-pool-docker-build--01234567-grp | 0           | true      | gke-project1234567-pool-docker-build--01234567-grp | 0          |
| gke-project12345-with-local-ssd-01234-12345678-grp | 0           | true      | gke-project12345-with-local-ssd-01234-12345678-grp | 0          |
| gke-project1234567-pool-with-local-ss-23456789-grp | 0           | true      | gke-project1234567-pool-with-local-ss-23456789-grp | 0          |
| gke-project12345-docker-build-0123456-34567890-grp | 0           | true      | gke-project12345-docker-build-0123456-34567890-grp | 0          |
| gke-tooling-service-pool-012345678910-45678901-grp | 2           | true      | gke-tooling-service-pool-012345678910-45678901-grp | 2          |
| gke-project1234567-onegit-service-poo-56789012-grp | 2           | true      | gke-project1234567-onegit-service-poo-56789012-grp | 2          |
| gke-project1234567-pool-spot-ssd-2345-67890123-grp | 4           | true      | gke-project1234567-pool-spot-ssd-2345-67890123-grp | 4          |
| gke-project12345-spot-ssd-01234567514-78901234-grp | 1           | <null>    | gke-project12345-spot-ssd-01234567514-78901234-grp | 1          |
+----------------------------------------------------+-------------+-----------+----------------------------------------------------+------------+
```
</details>

### CODE REVIEW

File: docs/tables/gcp_compute_instance_group_manager.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: "Steampipe Table: gcp_compute_instance_group_manager - Query Google Cloud Compute Engine Instance Group Managers using SQL"
+description: "Allows users to query Google Cloud Compute Engine Instance Group Managers, providing insights into the configuration, status, and properties of these group managers."
+---
+
+# Table: gcp_compute_instance_group_manager - Query Google Cloud Compute Engine Instance Group Managers using SQL
+
+Google Cloud Compute Engine Instance Group Managers manage [Managed Instance Groups (MIG)](https://cloud.google.com/compute/docs/instance-groups#managed_instance_groups). are ideal for highly available applications that require a lot of computing power and need to scale rapidly to meet demand. They offer a range of features including autoscaling, autohealing, regional (multiple zone) deployment, and automatic updating.
```
Comment (misraved) [inline docs/tables/gcp_compute_instance_group_manager.md:8]:
```suggestion
Google Cloud Compute Engine Instance Group Managers manage [Managed Instance Groups (MIG)](https://cloud.google.com/compute/docs/instance-groups#managed_instance_groups), and are ideal for highly available applications that require a lot of computing power and need to scale rapidly to meet demand. They offer a range of features including autoscaling, autohealing, regional (multiple zone) deployment, and automatic updating.
```

File: docs/tables/gcp_compute_instance_group_manager.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  description,
+  self_link,
+  instance_group,
+  location,
+  akas,
+  project
+from
+  gcp_compute_instance_group_manager;
+```
+
```
Comment (misraved) [inline docs/tables/gcp_compute_instance_group_manager.md:44]:
```suggestion
```

**Review by misraved — COMMENTED** [review]
@pdecat could you please take a look at the minor review comments 👍!!

Thanks once again for the new table 😄, great work!!

**Comment by pdecat** [comment]
Hi @misraved, thanks, I've applied your proposed changes!

**Review by misraved — APPROVED** [review]

**Pull request merged**